extensions/vscode/src/extension.ts

```diff
@@ -169,4 +170,19 @@ export const extensionSettings = {
     );
     return value !== undefined ? value : true;
   },
+  defaultConnectServer(): string {
+    const configuration = workspace.getConfiguration("positPublisher");
+    let value: string | undefined = configuration.get<string>(
+      "defaultConnectServer",
+    );
+
+    if (value === undefined || value === "") {
```

Comment:
Maybe add a comment explaining that this is to pull it from where Workbench puts it?

Reply:
Yes, good call on that comment.